signingscript/docker.d/passwords.yml

```diff
@@ -201,8 +201,8 @@ in:
              ["autograph_gpg"]
             ]
           - ["https://autograph-external.stage.autograph.services.mozaws.net",
-             {"$eval": "AUTOGRAPH_XPI_STAGE_USERNAME"},
-             {"$eval": "AUTOGRAPH_XPI_STAGE_PASSWORD"},
+             {"$eval": "AUTOGRAPH_STAGE_XPI_PRIVILEGED_USERNAME"},
+             {"$eval": "AUTOGRAPH_STAGE_XPI_PRIVILEGED_PASSWORD"},
```

Comment:
This is non-privileged signing, should it switch variables anyway?

Reply:
Good catch, I thought it was privileged but didn't notice the `cas_cur_extension` vs `ca_cur_webextension`